scripts/generate_user_email_list.py

```diff
@@ -1,67 +1,25 @@
 """
 
 Usage:
-    scripts/generate-user-email-list.py <data_api_url> <data_api_token> [--framework=<slug>]
+    scripts/generate-user-email-list.py <data_api_url> <data_api_token> [--framework=<slug>] [--status]
```

Comment:
Could we be more specific about what `status` is here - users also have the `active/deactivated` & `locked` statuses, but this is specifically the status of their declaration.

An example call in the comment, or list of possible values that `status` can take would be useful for future reference I think.